drud/ddev
# Fix #2886: replaces deprecated ioutils package.

## The Problem/Issue/Bug:
Fixes #2886.

## How this PR Solves The Problem:
Replaces the deprecated calls with corresponding calls from `io` or `os`.

## Manual Testing Instructions:
1. Restart an existing project and confirm you can run basic commands against it. 
2. Confirm you can create a new project.

## Automated Testing Overview:
No changes/additions to tests. There are no functional changes.

## Related Issue Link(s):
#2886

## Release/Deployment notes:
Improves compatibility with the latest versions of go.



<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/2911"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
